core/src/main/scala/akka/persistence/cassandra/journal/TagWriter.scala

```diff
@@ -253,15 +386,11 @@ import akka.util.UUIDComparator
         t)
       timers.startSingleTimer(FlushKey, InternalFlush, settings.flushInterval)
       parent ! TagWriters.TagWriteFailed(t)
-      context.become(idle(events ++ buffer, tagPidSequenceNrs))
+      context.become(idle(buffer, tagPidSequenceNrs))
```

Comment:
does the buffer still contain the `events` or is this that we don't retry them in this way any more?

Reply:
The buffer still contains the events, they are only removed once a writeComplete happens